rahuldkjain/github-profile-readme-generator
# feat: Update Gatsby to v3

## What type of PR is this? (check all applicable)


## Description

Hello, Gatsby maintainer here 👋 

I've updated your site to the current latest version of Gatsby so that you benefit from all the great things we shipped in v3. (v3 Release Notes: https://www.gatsbyjs.com/docs/reference/release-notes/v3.0)

Also note, that Gatsby v4 is on the horizon with some pretty cool features. Migrating from v3 to v4 will also be easier. So if you have the need for Server-Side Rendering or Deferred Static Generation, you can upgrade to v4: https://www.gatsbyjs.com/gatsby-4

Couple of improvements I also made:

- Moved your buymeacoffee widget to its proper lifecycle instead of modifying `html.js` directly
- Fixed a couple of ESLint warnings, including alt tag, links with `_blank` as target and a bug in the Twitter badge
- Set `purge: false` in tailwind since you use purgecss

Things you could upgrade in the future:

- Upgrade Tailwind to v2
- Use SVG images for all the images in src/images
- Update Jest (we have a Jest 27 guide here: https://www.gatsbyjs.com/docs/how-to/testing/unit-testing/)

## QA Instructions, Screenshots, Recordings

Browse the live site and check if everything works as expected. The local experience `gatsby develop` works as expected in my mind.
